evals/scabench/validate_score.py

```
@@ -0,0 +1,231 @@
+            for left_pick in itertools.combinations(lefts, size):
+                for right_pick in itertools.permutations(reports, size):
+                    if all(right_pick[i] in edge_map[left_pick[i]] for i in range(size)):
+                        found = True
+                        value = sum(w[(left_pick[i], right_pick[i])] for i in range(size))
+                        if (size, value) > best:
+                            best = (size, value)
+            if found:
+                break
+        return best
+
+    rng = random.Random(7)
```

Ruff (0.16.1)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)